templates/main/index.html.ep

```diff
@@ -44,6 +44,7 @@
             </div>
             <div class="form-group">
                 <strong>Misc</strong>
+                <label><input value="1" name="show_tags" type="checkbox" id="filter-show-tags"> Show tags</label>
```

Comment:
I will do the same as in: https://github.com/os-autoinst/openQA/pull/978

I have to check whether it works after this has been merged anyways.